### DISCUSSION TITLE
This project is still alive?

### DISCUSSION THREAD

**Discussion by idem2lyon** [diyhue/diyHue#723] [Q&A]
I found this project, seems very good..
But reading documentation, it appears that beta is still in beta for many month and no updates to stable...
There is no wiring schemas for the most importante, DIY lights !
I try to find wiring schemas for differents kinds of leds please.

Thanks

**Comment by mariusmotea** [comment]
If you take a look in commits sections you will see the project is alive and with lot of updates.

https://github.com/diyhue/diyHue/commits/beta

**Comment by juanesf** [comment]
Regarding connection diagrams, most of them are not necessary since there are many on the internet, the same connections from other projects are useful, you just have to change or verify the order and number of the connections. An example is wled which has projects for users that even on some devices comes pre-installed. In addition to being compatible with diyHue ...
